examples/ecs/ambiguity_checker.rs

```diff
@@ -0,0 +1,61 @@
+use bevy::{ecs::schedule::ReportExecutionOrderAmbiguities, log::LogPlugin, prelude::*};
+
+fn main() {
+    App::new()
+        .add_plugin(LogPlugin)
+        .insert_resource(MyStartupResource(0))
+        .insert_resource(1i32)
```

Comment:
Once #2254 is merged, integers will not be Components anymore.
This example should use a custom component, in preparation for that change.

Reply:
Thanks for pointing this out. Addressed that on https://github.com/bevyengine/bevy/pull/2766/commits/652b960f0623d51731610a2d9aaeb00806bf0cd7